### PR TITLE
✨ Add DayDetailsHeader

### LINKED ISSUE #9: As a User, I want to go to next/prev Day-Site with one Klick
- [x] Add next- and prev button to Day-Site
- [x] If the user clicks on the next page, but there is none, he lands on the first day page

### PR DESCRIPTION
**Pull request by m-wiemers**
Fix #9

### CODE REVIEW

File: components/dayDetailshead/DayDetailHead.module.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
+.container {
+  display: grid;
+  grid-template-columns: 25px auto 25px;
```
Comment (JuKahl) [inline components/dayDetailshead/DayDetailHead.module.css:3]: Have you tried "fr" instead of "px"?

**Review by JuKahl — APPROVED** [review]
GJ

**Review by fre-ben — APPROVED** [review]
LGTM

File: components/dayDetailshead/DayDetailHead.module.css (hunk 1 above)
Comment (m-wiemers) [reply components/dayDetailshead/DayDetailHead.module.css:3]: yes, i actually tried, but i couldn't get the next arrow to go to the end :-(

**Pull request merged**